.github/workflows/acceptance.yml

```diff
@@ -265,3 +341,13 @@ jobs:
 
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
+
+  fake-visual-snapshot:
```

Comment:
This workflow needs to stay here instead of visual-diff.yml in order that it will show up on the checks (rather than only being visible under the Actions tab)